sdk/src/object_crawler/fetching.rs

```diff
@@ -28,7 +28,7 @@ where
 }
 
 /// Fetch a dynamic field object from Sui based on the provided key.
-pub(crate) async fn dynamic_fetch_one<V>(
+pub async fn dynamic_fetch_one<V>(
```

Comment:
why is this changed? it's an internal function of the object crawler and should ideally stay that way